oauth2Flows.md

```diff
@@ -0,0 +1,25 @@
+# Flow diagrams
+## Create oauth2 client
+- Path: "/appid-<appId>/recipe/oauth2/client"
+- Method: POST 
+- Input:
+``` 
+{
+    redirectUris: string[],
+    name: string,
+    scopes: string[]
+}
+```
+- Output:
+```
+{
+    status : "OK",
+    clientId: string,
+    clientSecretBase64: string,
```

Comment:
why is this returned as base64? When querying the core with the secret, is this to be sent as base64?